samples/generic-base/monitoring/user-values-prom-operator.yaml

```diff
@@ -52,20 +52,20 @@
 #               storage: 25Gi
 
 # alertmanager:
-#   enabled:true
+#   enabled: true
 #   alertmanagerSpec:
 #     externalUrl: http://host.mycluster.example.com:31091
-#     storage:
-#       volumeClaimTemplate:
-#         spec:
-#           storageClassName: myCustomStorageClass
+#   storage:
```

Comment:
I think storage is a "child" of alertmanagerSpec and NOT of alertmanager.  That is how we are handling it in monitoring/values-prom-operator.yaml  (see [line 138 in that file](https://github.com/sassoftware/viya4-monitoring-kubernetes/blob/7fe23a8157f666c1c482dd2d13e88794745c58d6/monitoring/values-prom-operator.yaml#L138)).    And/or see [line 724 in values.yaml file in the Helm chart](https://github.com/prometheus-community/helm-charts/blob/5b2ac2b7c404bc14fe1bb42a5f96a4a45d7333f7/charts/kube-prometheus-stack/values.yaml#L724)

In addition, samples/generic-base/monitoring/user-values-prom-operator.yaml should generally match the contents of monitoring/values-prom-operator.yaml.  So, any change in one of those files should generally also be needed in the other one.  I notice you did not make this change in both files.